vela-templates/gen_definitions.sh

```diff
@@ -54,10 +55,28 @@ function render {
   fi
 }
 
+function renderMinimal {
+  inputDir=$1
+  outputDir=$2
+  rm "$outputDir"/* 2>/dev/null || true
+  mkdir -p "$outputDir"
+
+  $VELA_CMD def render "$inputDir" -o "$outputDir" --message "Definition source cue file: vela-templates/$inputDir/{{INPUT_FILENAME}}"
+  retVal=$?
+  if [ $retVal -ne 0 ]; then
+    echo -ne "${RED}Failed. Exit code: ${retVal}.${NC}\n"
+    exit $retVal
+  fi
+
+  rm -f $outputDir/env-binding.yaml
+  rm -f $outputDir/deploy2env.yaml  
+}
+
```

Comment:
This part is duplicated. Maybe we do not need to generate MINIMAL definitions but only need to copy them from INTERNAL.